data/json/items/ammo.json

```diff
@@ -946,6 +946,7 @@
         "category" : "chems",
         "price" : 3500,
         "name" : "sprayable fungicide",
+        "name_plural" : "sprayable fungicide",
```

Comment:
I'm not sure this is a real issue (at least I'm not able to reproduce it in ```master``` because fungicide is stored by-charges). Are you able to provide a snapshot from the game with this issue?